pyvelox/test/test_vector.py

```diff
@@ -351,3 +351,52 @@ def test_roundtrip_conversion(self):
                 self.assertTrue(velox_vector.dtype(), expected_type)
                 for i in range(0, len(data)):
                     self.assertEqual(velox_vector[i], data[i])
+
+    def test_row_vector_with_nulls(self):
+        vals = [
+            pv.from_list([1, 2, 3, 1, 2]),
+            pv.from_list([4, 5, 6, 4, 5]),
+            pv.from_list([7, 8, 9, 7, 8]),
+            pv.from_list([10, 11, 12, 10, 11]),
+        ]
+
+        col_names = ["a", "b", "c", "d"]
+        rw = pv.row_vector(col_names, vals, {0: True, 2: True})
+        rw_str = str(rw)
+        expected_str = (
+            "0: null\n1: {2, 5, 8, 11}\n2: null\n3: {1, 4, 7, 10}\n4: {2, 5, 8, 11}"
+        )
+        assert expected_str == rw_str
+
+    def test_row_vector_comparison(self):
```

Comment:
Can you add a test with different types (one stringview, one double, another ROW ) ?

Reply:
Sure, I will add.